RestApi/OData/FilterTranslator.g4

```diff
@@ -10,13 +10,12 @@ lexer grammar FilterTranslator;
 	public FilterTranslator(ICharStream input,
```

Comment:
❓ Is this constructor required for using this lexer? If so I can update the code to disallow the use of the other generated constructor.

Reply:
In this case yes because I want to provide some interfaces because lexer should check does identifier belongs to the table. Note that this is not classic lexer/parser - I just need lexer to transliterate tokens and it would be good just to do in-place validation for some tokens. In the bigger example I would have some real semantic analyzer, but this case I'm not generating some AST, so I just need some simple validation.